app/controllers/shipping_controller.rb

```diff
@@ -0,0 +1,43 @@
+require 'active_shipping'
+
+class ShippingController < ApplicationController
+  skip_before_filter :verify_authenticity_token
+
+  def ups_rates
+    ups = ActiveShipping::UPS.new(login: ENV['UPS_LOGIN'], password: ENV['UPS_PASSWORD'], key: ENV['UPS_ACCESS_KEY'])
+    ups_rates = get_rates(ups)
+    render :json => ups_rates, :status => (ups_rates.flatten.include?("error") ? 204 : 200)
```

Comment:
It would be great if you pass along with the status code a message to the user too. Lets say the user misspelled destination, and requested ups_rates. But it would give 204 status code giving no clue to the user what was wrong with the request.